README.md

```diff
@@ -306,33 +256,76 @@ Time: 0.016s
 postgres@localhost:postgres>
 ```
 
-##### Use case: Run database, database migrations, backend, and frontend locally
+## Testing
 
-Use the following to run the database, database migrations, backend server, and
-frontend client locally in docker.
+Run all tests other than Cypress in the project using `scripts/dev test`.
 
-```console
-COMPOSE_HTTP_TIMEOUT=120 docker-compose up --build
-```
+### Server tests
 
-Run the following to shut it down:
+Run `scripts/dev test:go`.
 
-```console
-docker-compose down
+### JS Tests
+
+Run `scripts/dev test:js`.
+
+### Cypress tests (End-to-end integration tests)
+
+There are multiple ways to run the Cypress tests:
+
+- Run `yarn run cypress run` to run the tests in the CLI. To have a slightly
+  more interactive experience, you can instead run `yarn run cypress open`.
+  Note: the database, frontend, and backend must be running prior to starting
+  the Cypress tests. The `APP_ENV` environment variable should be set to `test`.
+- `APP_ENV=test ./scripts/run-cypress-test-docker` : Run the Cypress tests,
+  database, migrations, backend, and frontend locally in Docker, similar to how
+  they run in CircleCI. Running the tests in this way takes time, but is useful
+  for troubleshooting integration test failures in CI.
+
+## Optional Setup
+
+### LaunchDarkly
+
+The app uses LaunchDarkly to control feature flags in deployed environments. By
+default the application run in offline mode and uses default values for all
+flags. To enable loading the flags from LaunchDarkly, add the following to
+`.envrc.local`:
+
+```bash
+export LD_SDK_KEY=sdk-0123456789
```

Comment:
It's unclear to me whether these are the values we're supposed to use, or if we are supposed to get the values from 1Password or LaunchDarkly

Reply:
Good point. There is a follow up story to add better docs for LD, but I can add a note about that here in the meantime.

Reply:
There's a sentence just after this that mentioned where to get them, and an upcoming story to ad more documentation for this as well.